rhc-ose-ansible/integration.yaml

```diff
@@ -0,0 +1,23 @@
+---
+- hosts: localhost
+  vars_prompt:
```

Comment:
@etsauer So this is where i'm slightly confused as i thought that the point was to not have an "inventory file"